modules/ingest-attachment/build.gradle

```diff
@@ -22,7 +22,15 @@ versions << [
   'mime4j': '0.8.5'
 ]
 
+// exclude commons-logging from test dependencies to avoid jar-hell, we use jcl-over-slf4j here
+configurations.testRuntimeClasspath { exclude module: 'commons-logging' }
+
 dependencies {
+  // take over logging for all dependencies, route slf4j and jcl over log4j
+  api "org.slf4j:slf4j-api:${versions.slf4j}"
```

Comment:
I'm not sure why slf4j is included in the global versions, but it is no longer a dependency of server. So this should really be a local version (so that it can match the version tika was compiled against).

Reply:
+1, fixed via #93755 which I've merged into this branch.